### PR TITLE
EPMRPP-78279 || Use React production build for reportportal.io

### PR DESCRIPTION
**Pull request by Vadim73i**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,9 +6,10 @@
   "scripts": {
     "lint": "eslint -f node_modules/eslint-friendly-formatter \"src/**/*.js\" --fix",
     "test": "",
-    "dev": "webpack serve --env dev",
+    "dev": "webpack serve",
+    "prod": "NODE_ENV='production' webpack serve",
```
Comment (chivekrodis) [inline package.json]: Do we need it?

Comment (Vadim73i) [reply package.json]: So we can test the production build locally

Comment (chivekrodis) [reply package.json]: I see, but looks like we need it only one time and for developers

Comment (AmsterGet) [reply package.json]: We don't use this script in production so it looks a bit confusing.
I would suggest to remove it.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -6,9 +6,10 @@
...
     "dev:local:documentation": "webpack serve --env proxyDocumentation --env dev",
-    "build": "webpack --env production"
+    "build": "NODE_ENV='production' webpack"
```
Comment (AmsterGet) [inline package.json]: What about other parts of webpack config where the `env.production` used? Should we refactor such places?

**Review by AmsterGet — CHANGES_REQUESTED** [review]

**Review by chivekrodis — APPROVED** [review]

**Comment by AmsterGet** [comment]
It looks like the new scripts don't work on Windows.
Don't merge until the changes are cross-platform.

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**